README.md

```diff
@@ -52,47 +52,48 @@ Read from https://github.com/erlang/otp/releases for each tag description as rel
-erlang     18.3            9f92145c1fac   10 days ago      754.3 MB
-erlang     18.3-slim       6d15a17b95d4   11 days ago      284.5 MB
+REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
+erlang              21.0-alpine         288ccf67e12b        7 hours ago         71.1MB
+erlang              21.0-slim           ebc139e56d88        7 hours ago         254MB
+erlang              21.0                76f0c5e69a48        7 hours ago         1.06GB
+erlang              20.3-alpine         4427af07b1fe        5 days ago          75.5MB
+erlang              20.3-slim           a5da071ec577        5 days ago          263MB
+erlang              20.3                b6e79a9bd68a        5 days ago          1.07GB
+erlang              18.3                91333142fa40        2 weeks ago         754MB
+erlang              19.3                2ea95f0c3147        2 weeks ago         886MB
+erlang              18.3-slim           f82b65021cfc        2 weeks ago         287MB
+erlang              19.3-slim           4f720aaec27c        2 weeks ago         440MB
 ```
 
 ### Running
 
 ```console
-$ docker run -it --rm erlang:20.0-rc1 /bin/bash
-root@11ecefc83eb5:/# ls /usr/local/lib/erlang/lib/
-asn1-5.0              dialyzer-3.2       kernel-5.3         sasl-3.0.4
-common_test-1.15      diameter-1.12.3    megaco-3.18.2      snmp-5.2.5
-compiler-7.1          edoc-0.9           mnesia-4.15        ssh-4.5
-cosEvent-2.2.1        eldap-1.2.2        observer-2.4       ssl-8.2
-cosEventDomain-1.2.1  erl_docgen-0.7     odbc-2.12          stdlib-3.4
-cosFileTransfer-1.2.1 erl_interface-3.10 orber-3.8.3        syntax_tools-2.1.2
-cosNotification-1.2.2 erts-9.0           os_mon-2.4.2       tools-2.10
-cosProperty-1.2.2     et-1.6             otp_mibs-1.1.1     wx-1.8.1
-cosTime-1.2.2         eunit-2.3.3        parsetools-2.1.5   xmerl-1.3.14
-cosTransactions-1.3.2 hipe-3.16          public_key-1.4.1
-crypto-4.0            ic-4.4.2           reltool-0.7.3
-debugger-4.2.2        inets-6.3.9        runtime_tools-1.12
-root@11ecefc83eb5:/# ls /usr/local/lib/erlang/lib/ | wc -l
-45
+$ docker run -it --rm erlang:21.0 /bin/bash
+root@ed434f6c1081:/# ls /usr/local/lib/erlang/lib/
+asn1-5.0.6        erl_interface-3.10.3  observer-2.8        ssh-4.7
+common_test-1.16  erts-10.0             odbc-2.12.1         ssl-9.0
+compiler-7.2      et-1.6.2              os_mon-2.4.5        stdlib-3.5
+crypto-4.3        eunit-2.3.6           otp_mibs-1.2        syntax_tools-2.1.5
+debugger-4.2.5    ftp-1.0               parsetools-2.1.7    tftp-1.0
+dialyzer-3.3      hipe-3.18             public_key-1.6      tools-3.0
+diameter-2.1.5    inets-7.0             reltool-0.7.6       wx-1.8.4
+edoc-0.9.3        kernel-6.0            runtime_tools-1.13  xmerl-1.3.17
+eldap-1.2.4       megaco-3.18.3         sasl-3.2
+erl_docgen-0.8    mnesia-4.15.4         snmp-5.2.11
+root@ed434f6c1081:/# ls /usr/local/lib/erlang/lib/ | wc -l
+     38      38     476 -
```

Comment:
did you actually run this line?  the `wc -l` is supposed to get number of lines only, not including number of words / number of characters;   should be one number only